BuildResidentialScheduleFile/resources/schedules.rb

```diff
@@ -371,7 +371,7 @@ def create_stochastic_schedules(args:,
     dw_minutes_between_event_gap = Constants.HotWaterDishwasherMinutesBetweenEventGap
     dw_activity_sch = [0] * mins_in_year
     m = 0
-    dw_flow_rate = gaussian_rand(prng, dw_flow_rate_mean, dw_flow_rate_std, 0)
+    dw_flow_rate = gaussian_rand(prng, dw_flow_rate_mean, dw_flow_rate_std, 0.1)
```

Comment:
I see that every call to `gaussian_rand` uses 0.1 for the final argument. Would it make more sense to update the method signature to use `max = 0.1` instead of `max = nil` and leave out the argument from all the callers?

Reply:
@rajeee?

Reply:
Yeah, can do that - though I don't have a strong feeling either way. Joe, your call.

Reply:
I'll go with suggestion from @shorowit.